apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-credit/style.scss

```diff
@@ -0,0 +1,37 @@
+.site-credit__block {
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	// @TODO: Pull these styles from the theme.
+	font-size: 14px;
```

Comment:
I think this is a great suggestion. Ideally, we would get these styles from the theme file -- I think there, it does use rem. However, when I copied that value here, it didn't display the same size. So I think the solution is to make sure theme styles are applying correctly here so that we include the rem from that